### ISSUE TITLE
improve handling of log information from installer script

### ISSUE THREAD

**Issue by vnepogodin** [CachyOS/CachyOS-Welcome#84]

**Comment by Aditya-PS-05** [comment]
Can you assign me this issue?

**Comment by ptr1337** [comment]
Just create a PR :) Thats fine.

**Comment by Aditya-PS-05** [comment]
Is `cachyos-hello.desktop` this file where I need to make the changement. 

One request, can we make a contributions.md file to help the new contributors to help.

**Comment by Aditya-PS-05** [comment]
Is `cachyos-hello.desktop` the file where I need to make changes?

One request, can we make a discussion forem to help new contributors to how to setup the repos as there are various build issues.

**Comment by NextWork123** [comment]
> Is `cachyos-hello.desktop` the file where I need to make changes?

I don't mean to be rude, but what exactly do you want to modify in `cachyos-hello.desktop` if it’s just a shortcut? It seems the root of this issue might be elsewhere. Maybe just mention that there are multiple locations handled by that file, like in the `cachyos-calamares online script` and in `hello`

**Comment by vnepogodin** [comment]
> Is `cachyos-hello.desktop` the file where I need to make changes?
> 
> One request, can we make a discussion forem to help new contributors to how to setup the repos as there are various build issues.

No. I meant that the cachyos when it spawns installer script it needs to write the log into file instead of directly writing into stdout.

Logging only installer script output is dumb, and so whole cachyos-welcome needs to use proper logger

**Comment by vnepogodin** [comment]
fixed in https://github.com/CachyOS/CachyOS-Welcome/commit/1d99e072e30a86f81dcec9511bc5c84772da9e5a
